scss/sideBar.module.scss

```diff
@@ -0,0 +1,144 @@
+$primary-color: hsl(0, 0%, 98.4%);
+
+.sidebar_title {
+  width: 327px;
+  height: 22px;
+  left: 0px;
+  top: 0px;
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 22px;
+  text-align: center;
+  color: #000000;
+  margin-bottom: 22px;
+}
+
+.sidebar_mainbody {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 28px 21px;
+
+  position: static;
```

Comment:
`position, left, top, flex, order, flex-grow` should be removed. width and height could turn into:

```css
width: 100%;
max-width: 369px;
height: 100%;
max-height: 460px;
```